tests/test_aiosource.py

```diff
@@ -18,32 +18,21 @@
 
 """Tests for package index handling - package source control."""
 
-import os
-import json
-import typing
-
-from pathlib import Path
-
 import pytest
-import requests
-
-from flexmock import flexmock
-
-from thoth.python.aiosource import AIOSource, AsyncIterablePackages, AsyncIterableVersions
-from thoth.python.artifact import Artifact
 
-from .base import PythonTestCase
+from thoth.python.aiosource import AIOSource, AsyncIterablePackages
 
-import tempfile
-
-from zipfile import ZipFile
+from base import PythonTestCase
```

Comment:
```suggestion
from tests.base import PythonTestCase
```
base is not a packages, as it a file from which we are importing, if we dont want to use `.base` then please use absolute path `tests.base`.
this is causing some of the tests to fail. please fix this in other test scripts.

Reply:
did them